### PR TITLE
fix(Upload): The file name is too long bug

### PR DESCRIPTION
**Pull request by jiaowoxiaobala**
### `General`

> ✏️ Mark the necessary items without changing the structure of the PR template.

- [ ] Pull request template structure not broken

### `Type`

> ℹ️ What types of changes does your code introduce?

> 👉 _Put an `x` in the boxes that apply_

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### `Checklist`

> ℹ️ Check all checkboxes - this will indicate that you have done everything in accordance with the rules in [CONTRIBUTING](contributing.md).

> 👉 _Put an `x` in the boxes that apply._

- [x] My code follows the style guidelines of this project
- [x] Is the code format correct
- [x] Is the git submission information standard?
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by jiaowoxiaobala** [comment]
#1355 #1855

**Comment by jiaowoxiaobala** [comment]
Issues #1355 Issues #1855

File: src/locales/lang/en/component.json [hunk 1]
```diff
@@ -107,6 +107,7 @@
     "fileName": "File name", 
     "fileSize": "File size", 
     "fileStatue": "File status", 
+    "noUpload": "No Upload",
```
Comment (likui628) [inline src/locales/lang/en/component.json:110]: 待上传用No Upload不太合适

Comment (jiaowoxiaobala) [reply src/locales/lang/en/component.json]: @likui628  ok

**Comment by likui628** [comment]
@jiaowoxiaobala 我把待上传改为pending了，感觉这个更合适。
顺便修了下ts类型

**Pull request merged**